### ISSUE TITLE
Tensorboard tutorial add_pr_curve( )

### ISSUE THREAD

**Issue by kanru-wang** [pytorch/tutorials#852]
In [https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html#](url), section 6, Assessing trained models with TensorBoard, there are these two lines.

        class_probs.append(class_probs_batch)
        class_preds.append(class_preds_batch)

Shouldn't the second line be 

        class_preds.append(labels)

instead? 

because later `add_pr_curve` takes ground truth labels instead of predicted labels.

**Comment by WYBupup** [comment]
I have found this problem, too. And I change the second parameter of writer.add_pr_curve() from “test_preds == class_index” to “testset.targets == class_index”. And I got terrible percision-Recall curve for each class( one of the pr_curve was showed in the following picture). I just wonder if you have corrected the error in the tutorial and run the code you corrected. If you have, was the pr curve you got same as mine?
I am looking forward to your "response!"
![PR_curve_corrected_code](https://user-images.githubusercontent.com/61353633/76382156-bfa72a00-6392-11ea-8f24-f9e750e45ddd.png)

**Comment by WYBupup** [comment]
I have found this problem, too. And I change the second parameter of writer.add_pr_curve() from “test_preds == class_index” to “testset.targets == class_index”. And I got terrible percision-Recall curve for each class( one of the pr_curve was showed in the following picture). I just wonder if you have corrected the error in the tutorial and run the code you corrected. If you have, was the pr curve you got same as mine?
I am looking forward to your "response!"
![PR_curve_corrected_code](https://user-images.githubusercontent.com/61353633/76382223-efeec880-6392-11ea-8c59-778d995f136c.png)

**Comment by WYBupup** [comment]
As for the terrible result of pr_curve I got after I changing “test_preds==class_index” to “test_preds == testset.targets”, the reason is the order of testset.targets doesn’t match tensorboard_probs’ , since the later one was got from testloader which was created by shuffling the testset. Instead, I changed the code “class_preds.append(class_preds_batch)” to “class_preds.append(labels)” like what you said. I got pretty reasonable pr_curve.(showed in the picture)
![pr_curve_v2](https://user-images.githubusercontent.com/61353633/76392292-36045600-63ac-11ea-9e44-04406f967a45.png)

**Comment by dwang55** [comment]
Guys, I found the same thing. You crroect the code the same way just as me. I didn't check the github, so I opend a topic on Pytorch forum before. 
It should be a code error in the tutorial, thank you for make me sure for that.

**Comment by holly1238** [comment]
This issue is fixed and merged in https://github.com/pytorch/tutorials/pull/1469
